plugins/woocommerce/src/Blocks/BlockTypes/MiniCart.php

```diff
@@ -418,8 +418,8 @@ protected function get_markup( $attributes ) {
 		$wrapper_classes = sprintf( 'wc-block-mini-cart wp-block-woocommerce-mini-cart %s', $classes_styles['classes'] );
 		$wrapper_styles  = $classes_styles['styles'];
 
-		$icon_color          = array_key_exists( 'iconColor', $attributes ) ? esc_attr( $attributes['iconColor']['color'] ) : 'currentColor';
-		$product_count_color = array_key_exists( 'productCountColor', $attributes ) ? esc_attr( $attributes['productCountColor']['color'] ) : '';
+		$icon_color          = $attributes['iconColor']['color'] ?? 'currentcolor';
+		$product_count_color = $attributes['productCountColor']['color'] ?? '';
```

Comment:
`$product_count_color` needs to be escaped as it goes inside a html attribute.

Reply:
Thanks, resolved.